draft-johani-dnsop-multi-provider-requirements-00.md

```diff
@@ -148,6 +148,199 @@ to be able to fully support all multi-provider scenarios:
    DNSSEC signer" is defined as a bump-on-the-wire DNSSEC signer with
    support for multi-signer key rollovers.
 
+# Scenario-based analysis
+
+This section describes a system of analyzing scenarios with respect to
+zone contents and then derive requirements from the generation and
+modifications of those zones. The idea is that any mechanism can be linked
+through the requirements to particular contents of changes to the contents
+of a zone that are needed.
+
+## Scenarios
+
+This subsection lists the scenarios that are analyzed in order from low
+to high complexity.
+
+1) Simple multi-signer
+
+   This scenario consists of two providers A and B that each both sign and
+   provide the zone. Both providers receive an unsigned zone from the zone
+   owner. The parent has a CDS/CDNSKEY/CSYNC scanner.
+
+(More scenarios are needed)
+
+## Simple multi-signer Scenario
+
+This subsection analyses the simple multi-signer scenario
+
+### Zone contents
+
+Lets assume that each signer has a KSK and a ZSK called KSK_A, ZSK_A, KSK_B,
+and ZSK_B. Each provider has NS records, NS_A for provider A and NS_B for
+provider B.
+
+The quiescent state is the following:
+
+Provider A signs the zone (except for the DNSKEY, CDS and CDNSKEY RRsets) with
+ZSK_A. Provider B signs with ZSK_B. Provider A's DNSKEY RRset contains at 
+least KSK_A, ZSK_A, and ZSK_B. And is signed by at least KSK_A. Similarly,
+provider B's DNSKEY RRset contains at least KSK_B, ZSK_A, and ZSK_B. This
+RRset is signed by KSK_B. Both providers have an NS RRset that consists of
+NS_A and NS_B. At the parent the DS records refer to KSK_A and KSK_B.
+
+### ZSK rolls
+
+RFC 6781 describes two ZSK rolls, a Pre-Publish ZSK roll and a
+Double-Signature ZSK roll. In a multi-signer environment, the Double-Signature
+ZSK roll has fewer synchronization requirements, but is generally undesirable
+because it leads to more signatures. We will first look at the Double-Signature
+ZSK roll.
```

Comment:
I would think that the multi-signer coordination protocol doesn't care which rollover flavor it is: during a ZSK roll, one provider switches to a *new ZSK set*. Depending on the flavor, this has to happen once or twice, but the way how the coordination is done can be identical.

Reply:
I agree with Peter. While the multi-signer rollover details are needed somewhere, the requirements document is not the place.